adflow/pyADflow.py

```diff
@@ -1387,6 +1389,230 @@ def evalFunctionsSens(self, aeroProblem, funcsSens, evalFuncs=None):
             print('| %-30s: %10.3f sec'%('Complete Sensitivity Time',finalEvalSensTime - startEvalSensTime))
             print('+--------------------------------------------------+')
 
+    def evalFunctionsSensFwd(self, aeroProblem, funcsSens, evalFuncs=None):
```

Comment:
we didnt have time to review this function, but we should go through this in detail and update comments as necessary. we also need to get the pygeo working with this to test, so there is potentially a bit of development that needs to be done. I think you should put these changes in a separate branch since they are very localized, and we can work on this separately. 

Also, this stuff likely does not have any effect on the rotation stuff, and this function is purely python to check the forward mode AD routines with bwd mode ones.

Reply:
Agreed on this, I can open a PR on this issue soon